as_perimenoun_oi_gynaikes.ass

```diff
@@ -1096,18 +1097,19 @@ Dialogue: 0,1:18:28.01,1:18:32.56,Default,,0,0,0,,{\i1}Don't feel guilt for what
 Dialogue: 0,1:18:32.56,1:18:37.66,Default,,0,0,0,,{\i1}All our mistakes are mine.{\i0}
 Dialogue: 0,1:18:53.69,1:18:56.01,Default,,0,0,0,,Finally! It's noon!
 Dialogue: 0,1:18:56.01,1:18:58.31,Default,,0,0,0,,I had to gather our things alone and it's still my fault!?
-Dialogue: 0,1:18:58.31,1:19:00.49,Default,,0,0,0,,So, I'm not allowed to have back-pain this one time?
+Dialogue: 0,1:18:58.31,1:19:00.49,Default,,0,0,0,,So, We are not allowed to have back-pain this one time?
 Dialogue: 0,1:19:00.49,1:19:01.81,Default,,0,0,0,,Must we fight?
 Dialogue: 0,1:19:01.81,1:19:03.81,Default,,0,0,0,,I really don't get him sometimes...
-Dialogue: 0,1:19:08.59,1:19:13.68,Default,,0,0,0,,Ms. Archontoula, Ms. Soultana, we will come and see you once a week at the least.
+Dialogue: 0,1:19:08.59,1:19:13.68,Default,,0,0,0,,Ms. Archontoula, Ms. Soultana, we will come and see you once a week minimum.
 Dialogue: 0,1:19:13.68,1:19:15.21,Default,,0,0,0,,Panos, Antonis, right?
 Dialogue: 0,1:19:15.21,1:19:17.21,Default,,0,0,0,,But of course.
 Dialogue: 0,1:19:24.21,1:19:28.96,Default,,0,0,0,,Visit again Mr. Antonis! Visit again Mr. Panos!
+Dialogue: 0,1:19:29.00,1:19:32.02,Default,,0,0,0,,We will come Ms. Soultana
 Dialogue: 0,1:20:29.16,1:20:30.84,Default,,0,0,0,,- What is up?\N- What is up?
 Dialogue: 0,1:20:30.84,1:20:33.44,Default,,0,0,0,,- Why did we stop?\N- Antonis did.
 Dialogue: 0,1:20:33.44,1:20:35.44,Default,,0,0,0,,Michalis, we have to stay here a bit and recuperate.
 Dialogue: 0,1:20:35.44,1:20:36.76,Default,,0,0,0,,And take a swim too.
-Dialogue: 0,1:20:36.76,1:20:38.89,Default,,0,0,0,,So that we don't catch the heat in the ferry.
+Dialogue: 0,1:20:36.76,1:20:38.89,Default,,0,0,0,,So that we avoid noun's heat in the ferry.

```

Comment:
Syntactically wrong and does not improve on the original translation.

Reply:
Catch heat in English has a different meaning. avoid heat is better in this context.

https://en.wiktionary.org/wiki/catch_heat

Reply:
OK, let's make it "So that we avoid the heat in the ferry.".